crypto/mem_sec.c

```diff
@@ -31,6 +31,10 @@
 # include <unistd.h>
 # include <sys/types.h>
 # include <sys/mman.h>
+# if defined(OPENSSL_SYS_LINUX)
+#  include <sys/syscall.h>
+#  include <linux/mman.h>
```

Comment:
Yes, I explicitly checked Ubuntu 16, 14, Fedora 25, and looked all the way back to 2.6.13 on lxr.linux.no. Also linux/mman.h does have multiple-inclusion protection.